etc/openstack-config/openstack-config.yml

```diff
@@ -399,116 +369,93 @@ hpc_v1_32cpu:
     hw:numa_nodes: 8
     hw:pci_numa_affinity_policy: preferred
     hw_rng:allowed: "True"
+    trait:CUSTOM_COMPUTE_STANDARD: "required"
+    resources:CUSTOM_COMPUTE_STANDARD: 2
 
-hpc_v1_48cpu:
-  name: "hpc.v1.48cpu"
-  ram: 196608
+# HPC v2:
+# Core-pinned VCPUs
+# 8GB RAM per VCPU
+# Spread across NUMA regions
+# Slurm highmem compute node
+hpc_v2_60cpu_480ram:
+  name: "hpc.v2.60cpu.480ram"
+  ram: 491520
   disk: 30
-  vcpus: 48
+  vcpus: 60
   is_public: false
   extra_specs:
     hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:cpu_threads: 2
     hw:mem_page_size: "1GB"
     hw:cpu_sockets: 2
     hw:numa_nodes: 8
     hw:pci_numa_affinity_policy: preferred
     hw_rng:allowed: "True"
+    trait:CUSTOM_COMPUTE_HIGHMEM: "required"
+    resources:CUSTOM_COMPUTE_HIGHMEM: 1
 
-hpc_v1_64cpu:
-  name: "hpc.v1.64cpu"
-  ram: 262144
+# HPC v2:
+# Core-pinned VCPUs
+# 8GB RAM per VCPU
```

Comment:
8 GB per CPU but also used on hyperthreaded CUSTOM_COMPUTE_STANDARD which means we have inconsistent ram to cpu ratio (Should have a 4X ratio).  This does have the advantage of not splitting the flavors though. I think this means we end up not using 8 of the 240 cores that we earmarked for VMs.